src/QirRuntime/test/FullstateSimulator/qsharp/qir-test-simulator.qs

```diff
@@ -33,49 +33,49 @@ namespace Microsoft.Quantum.Testing.QIR
     operation Test_Simulator_QIS() : Int
     {
         mutable res = 0;
-        let res = InvokeAllVariants(X);
+        set res = InvokeAllVariants(X);
         if (res != 0) { return res; }
 
-        let res = InvokeAllVariants(Y);
+        set res = InvokeAllVariants(Y);
         if (res != 0) { return 10 + res; }
 
-        let res = InvokeAllVariants(Z);
+        set res = InvokeAllVariants(Z);
         if (res != 0) { return 20 + res; }
 
-        let res = InvokeAllVariants(H);
+        set res = InvokeAllVariants(H);
         if (res != 0) { return 30 + res; }
 
-        let res = InvokeAllVariants(S);
+        set res = InvokeAllVariants(S);
         if (res != 0) { return 40 + res; }
 
-        let res = InvokeAllVariants(T);
+        set res = InvokeAllVariants(T);
         if (res != 0) { return 50 + res; }
 
-        let res = InvokeAllVariants(R(PauliX, 0.42, _));
+        set res = InvokeAllVariants(R(PauliX, 0.42, _));
         if (res != 0) { return 60 + res; }
 
-        using((targets, ctls) = (Qubit[2], Qubit[2]))
+        use (targets, ctls) = (Qubit[2], Qubit[2])
         {
             let theta = 0.42;
             Exp([PauliX, PauliY], theta, targets);
             Adjoint Exp([PauliX, PauliY], theta, targets);
-            if (M(target) != Zero) { let res = 1; }
+            if (M(targets[0]) != Zero) { set res = 1; }
```

Comment:
As you pointed out in my change, this probably also include putting the rest of the scope body into an else clause.

Reply:
Fixed. Thanks ;-)